app/models/couch.rb

```diff
@@ -4,7 +4,7 @@ class Couch < ApplicationRecord
   has_many :nights
   has_many :reservations, through: :nights
 
-  scope :in_city, -> (city) { where(city: city) }
+  scope :in_city, -> (city) { where("lower(city) = ?", city.downcase) }
```

Comment:
That's a good thought, @njgheorghita.  From what we've built out so far, it seems like mostly we'll be clicking on the couch to reserve it, and less likely to be searching for the couch named, "Big Bertha"... until we can vouch for couches and people want to stay on the famous ones that is...

Until then, city search looks good!  Thanks @tmikeschu